### PR TITLE
Feature/day 3

### PR DESCRIPTION
**Pull request by ayushpandey-successive**
Programs based on Loops and functions

1. Write a program to perform functionality of a calculator (add,sub,multiply,divide)
question1

2. Write a program to display following output as shown in figure

1
2 3
4 5 6
7 8 9 10

1
2 2
3 3 3
4 4 4 4

1 2 3 4 5
1 2 3 4
1 2 3
1 2
1
1 2
1 2 3
1 2 3 4
1 2 3 4 5

ques2

3. Write a program to display Diamond Pattern
ques3

4. Write a program to print all even number first and then all odd numbers
ques5

5. Write a program to print all even number first and then all odd numbers using only one iteration
ques5

6. Write a program to display prime numbers from 1 to 50
ques6

7. Write a program to display the twice of a number starting from 2 and end at 4096 (2, 4, 8, 16, ..... 4096)
ques7

8. Write a program to display number from 1-20 using all types of loops
ques8

9. Write a program to print fibonaaci series
ques9

### CODE REVIEW

File: day-3/ques-1.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+//1. Write a program to perform functionality of a calculator (add,sub,multiply,divide)
+
+
+const calc=(a,b,op)=>{
```
Comment (saurabh-yadav-successive) [inline day-3/ques-1.js]: variable name should be meaningful.

File: day-3/ques-1.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    if(op=="+")
```
Comment (saurabh-yadav-successive) [inline day-3/ques-1.js]: use switch case instead od if else condition

File: day-3/ques-2.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+// 1
+// 2 3
+// 4 5 6
+// 7 8 9 10
+let range=4;
```
Comment (saurabh-yadav-successive) [inline day-3/ques-2.js]: remove unnecessary let

File: day-3/ques-2.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+let z=1;
+for(let i = 0; i <range;i++)
```
Comment (saurabh-yadav-successive) [inline day-3/ques-2.js]: correct the code format

File: day-3/ques-2.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+{
+    let x="";
+    for(let j = 0; j <i+1; j++)
+    {
+        //console.log(i+j+1);
+        x=x+(z)+" ";
```
Comment (saurabh-yadav-successive) [inline day-3/ques-2.js]: remove unnecessary brackets. and comments

File: day-3/ques-2.js [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+        z++;
+    }
+    console.log(x);
+    
+}
+
+console.log();
```
Comment (saurabh-yadav-successive) [inline day-3/ques-2.js:20]: remove unnecessary logs

File: day-3/ques-2.js [hunk 7]
```diff
@@ -0,0 +1,83 @@
+// 1
+// 2 3
+// 4 5 6
+// 7 8 9 10
```
Comment (saurabh-yadav-successive) [inline day-3/ques-2.js:4]: create it just like a function which takes input as a range and starting value.
After that IT creates pyramid accordingly.

File: day-3/ques-2.js [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+// 1
+// 2 2
+// 3 3 3
+// 4 4 4 4
+
+
```
Comment (saurabh-yadav-successive) [inline day-3/ques-2.js]: make it function which take r value and create pyramid accordingly.

File: day-3/ques-2.js [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+let r=4;
```
Comment (saurabh-yadav-successive) [inline day-3/ques-2.js]: remove unnecessary let, commented code and logs

File: day-3/ques-2.js [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+for(let i = 1; i <r+1;i++)
```
Comment (saurabh-yadav-successive) [inline day-3/ques-2.js]: code format

File: day-3/ques-2.js [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+// 1 2 3 4
+// 1 2 3
+// 1 2
+// 1
+// 1 2
+// 1 2 3
+// 1 2 3 4
+// 1 2 3 4 5
+
+let m=5;
```
Comment (saurabh-yadav-successive) [inline day-3/ques-2.js]: follow above comments

**Review by saurabh-yadav-successive — CHANGES_REQUESTED** [review]

File: day-3/ques-1.js (hunk 1 above)
Comment (ayushpandey-successive) [reply day-3/ques-1.js]: resolved

File: day-3/ques-1.js (hunk 2 above)
Comment (ayushpandey-successive) [reply day-3/ques-1.js]: resolved

File: day-3/ques-2.js (hunk 3 above)
Comment (ayushpandey-successive) [reply day-3/ques-2.js]: resolved

File: day-3/ques-2.js (hunk 4 above)
Comment (ayushpandey-successive) [reply day-3/ques-2.js]: resolved

File: day-3/ques-2.js (hunk 5 above)
Comment (ayushpandey-successive) [reply day-3/ques-2.js]: resolved

File: day-3/ques-2.js (hunk 6 above)
Comment (ayushpandey-successive) [reply day-3/ques-2.js:20]: This console.log() is to separate the patterns

File: day-3/ques-2.js (hunk 7 above)
Comment (ayushpandey-successive) [reply day-3/ques-2.js:4]: resolved

File: day-3/ques-2.js (hunk 8 above)
Comment (ayushpandey-successive) [reply day-3/ques-2.js]: resolved

File: day-3/ques-2.js (hunk 9 above)
Comment (ayushpandey-successive) [reply day-3/ques-2.js]: resolved

File: day-3/ques-2.js (hunk 10 above)
Comment (ayushpandey-successive) [reply day-3/ques-2.js]: resolved

File: day-3/ques-2.js (hunk 11 above)
Comment (ayushpandey-successive) [reply day-3/ques-2.js]: resolved

**Review by saurabh-yadav-successive — APPROVED** [review]

**Pull request merged**